src/globals/tabnineExtensionProperties.ts

```diff
@@ -175,6 +181,14 @@ function getContext(): TabNineExtensionProperties {
       }
       return undefined;
     },
+    get shouldRunBootstrap(): boolean {
+      return !cloudHost;
```

Comment:
I think it's better to do that based if enterprise, not if cloudHost is configured